{
    "type": "bundle",
    "id": "bundle--683c9bcd-d56a-4159-8749-9284806a5f61",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--a54a96a3-0c79-4794-bca5-e275f7a459d7",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:12:08.232515Z",
            "modified": "2021-03-13T20:12:08.232515Z",
            "name": "CVE-2019-5729",
            "description": "Splunk-SDK-Python before 1.6.6 does not properly verify untrusted TLS server certificates, which could result in man-in-the-middle attacks.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2019-5729"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}